M2/Macaulay2/packages/Macaulay2Doc/doc4.m2

```diff
@@ -133,6 +133,19 @@ document {
      }
 
 document {
+     Key => {(pairs, Sequence),(pairs, List)},
```

Comment:
The key should be simply (pairs,BasicList), because "List" in our "d"
code corresponds to "BasicList" at top level, and List and Sequence
or both types of BasicList.